lua/telescope/builtin/files.lua

```diff
@@ -202,8 +203,11 @@ files.find_files = function(opts)
     if follow then
       table.insert(find_command, "-L")
     end
+    if search_file then
+      table.insert(find_command, search_file)
+    end
     if search_dirs then
-      if command ~= "rg" then
+      if command ~= "rg" and not search_file then
```

Comment:
you can always define your own find_command. For me right now this would be just a short_hand option. Right now you can already do `lua require("telescope.builtin").find_files { find_command = { "fd", "--type", "f", "asdf" } }`

With fzf-native you can add a space after keymaps to start a new search term and then it should work.

I am fine with merging. I fix the conflict for you. But I still think it doesnt work with rg. Because this new option is for searching a string but if i do the following i get `asdf` does not exist

![image](https://user-images.githubusercontent.com/15233006/176933259-047c9b3c-d0ba-4b46-9169-7bed6bb27365.png)

And i am on the latest rg version pretty sure. If there is a unreleased commit that adds support for it? or do i do something wrong

Reply:
Oh great! A custom command would also do the trick. Thank you!

You are right. It's not working with rg. For rg it would be neccecary to pass `rg --files -g '*asdf*' ` or `rg --files | rg asdf`.
Which I both failed to get working with telescope.

Reply:
second one doesnt work because we dont have a bash interpreter, or similar that can actually understand pipes, we have to do that ourself. Of course your command can start with `bash -c` but that isnt great.

First one should be `find_command = { "rg", "--files", "-g", "*asdf*" }` works for me

Reply:
If you want this merged, it has to be today, we somewhat have a feature freeze today see #1938 

Again i would merge if it is fully implemented :+1: